tests/js/spec/views/eventsV2/table/columnEditModal.spec.js

```diff
@@ -357,6 +357,40 @@ describe('EventsV2 -> ColumnEditModal', function () {
         'Maximum operators exceeded'
       );
     });
+
+    it('resets required field to previous value if cleared', function () {
+      const initialColumnVal = '0.6';
+      const newWrapper = mountModal(
+        {
+          columns: [
+            {
+              kind: 'function',
+              function: ['percentile', 'transaction.duration', initialColumnVal],
+            },
+          ],
+          onApply: () => void 0,
```

Comment:
hmm, since you're doing this, onApply on L387 might not be picked up correctly? I think you might have to define something like `const onApply = jest.fn();` first, pass that here to have L387 pick up the correct function call

Reply:
Removed the function to use the mocked `onApply` from the `beforeEach`. Thanks for catching that!